expo-boomboomapp/src/services/UserService/userServiceI.ts

```diff
@@ -16,15 +16,22 @@ export type UserI = {
 export type UserState = UserStateConnected | UserStateNotConnected;
 
 export type UserStateConnected = {
-    isConnected: true;
+  isConnected: true;
 } & UserI;
 
 export type UserStateNotConnected = {
-    isConnected: false;
+  isConnected: false;
 };
 
 export enum Gender {
   MALE = 1,
   FEMALE = 2,
   NO_SPECIFIC = 3,
 }
+
+export const GenderTextMapping: Record<Gender, string> = {
+  // TODO : I18n
+  [Gender.MALE]: 'Male',
```

Comment:
Use I18n keys here, you can update keys in fr.ts and en.ts ;)

Reply:
Finally I'll remove it !